src/VisualStudio/Core/Def/Implementation/ProjectSystem/AbstractProject.cs

```diff
@@ -1007,6 +970,8 @@ public virtual void Disconnect()
                 this.ProjectTracker.RemoveProject(this);
 
                 _pushingChangesToWorkspaceHosts = false;
+
+                this.EditAndContinueImplOpt = null;
```

Comment:
I have retained the [current references/assignments](http://source.roslyn.io/#Microsoft.VisualStudio.LanguageServices/Implementation/ProjectSystem/AbstractRoslynProject.cs,1b75062494c3b88d,references) to it, just moved it out from AbstractRoslynProject down to AbstractProject.